src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -0,0 +1,400 @@
+.. :changelog:
```

Comment:
Please do not include this file. Rather, the changes to the release notes that you'd like to make should bee added as a section in [the broader `azure-cli`'s release notes](https://github.com/Azure/azure-cli/blob/7c5f44151010b4b64d822f8cbe3e725f3525a448/src/azure-cli/HISTORY.rst#release-history).